backend/src/routes/v0/rag/__init__.py

```
@@ -0,0 +1,205 @@
+		responses={
+			status.HTTP_200_OK: CLIENT_SPEC["paths"]["/collections/{collection_id}/documents"]["post"]["responses"]["200"],
+		},
+		summary=CLIENT_SPEC["paths"]["/collections/{collection_id}/documents"]["post"]["summary"],
+		description=CLIENT_SPEC["paths"]["/collections/{collection_id}/documents"]["post"]["description"],
+		operation_id=CLIENT_SPEC["paths"]["/collections/{collection_id}/documents"]["post"]["operationId"],
+		response_model=dict[str, Any],
+	)
+	async def documents_create(
+		request: Request, 
+		collection_id: UUID,
+		files: list[UploadFile] = File(...),
```

Ruff (0.11.9)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)